File: lab8/sort.py
def bubbleSort(array):
   n = len(array)
   swapped = True
   temp = 0
   while (swapped == True):
      swapped = False
      for i in range(1,n-1,1):
         if array[i-1]>array[i]:
            temp = array[i-1]
            array[i-1]=array[i]
            array[i]= temp
            swapped = True
   status = True
   return [status, array]

File: lab8/sort.c
#include<stdio.h>

int bubbleSort(int *array, int size){
   int n = size;
   int swapped = 1;
   int i; 
   int temp;
   if (array == NULL){
      return -1;
   }
   while (swapped == 1){
      swapped = 0;
      for (i=1; i<n; i++){
         if (array[i-1]>array[i]){
            temp = array[i-1];
            array[i-1] = array[i];
            array[i]=temp;
            swapped = 1;
         }
      }
   }
   return 0;
}


